pkg/network/dhcp/serverv6/values.go

```diff
@@ -0,0 +1,7 @@
+package serverv6
+
+import "time"
+
+const (
+	infiniteLease = 999 * 24 * time.Hour
```

Comment:
This is strange... Why it is not defined near the usage or the most appropriate source file? It is strange to have a generic file named "values".

Reply:
I still see this fine in commit `net, dhcp: Move IPv6 server to serverv6 package`